packages/semantic-ui-theme/semantic/src/definitions/modules/dropdown.less

```diff
@@ -563,10 +563,11 @@ select.ui.dropdown {
   position: absolute;
   z-index: 2;
 }
+
 .ui.search.dropdown > .text {
   cursor: text;
   position: relative;
-  left: @textCursorSpacing;
```

Comment:
if there is a variable, can't we instead update the value of the variable in sharegate theme?